python/dgl/distributed/__init__.py

```diff
@@ -6,6 +6,8 @@
     DistDataLoader,
     DistEdgeDataLoader,
     DistNodeDataLoader,
+    EdgeCollator,
+    NodeCollator,
```

Comment:
Do we still need the list if we have __all__?

Reply:
double assurance. let's avoid using `import *`